### PR TITLE
EDURED107

### PR DESCRIPTION
**Pull request by risinggalaxy**
The first registering flows are ready for testing

### CODE REVIEW

File: Sources/EduIDExpansion/Flows/CreateEduID/ViewModels/CreatePincodeAndBiometricAccessViewModel.swift [hunk 1]
```diff
@@ -74,29 +77,40 @@ final class CreatePincodeAndBiometricAccessViewModel: NSObject {
...
+                        .execute()
+                        .body
+                    
+                    ServiceContainer.sharedInstance().challengeService.startChallenge(fromScanResult: enrolment.url ?? "") { [weak self] type, object, error in
+                        guard let self else { return }
+                        self.secondEnteredPin.removeLast(2)
+                        self.createIdentity(for: object as? EnrollmentChallenge, completion: completion)
                     }
+                } catch let error as NSError {
+                    dump(error)
```
Comment (dzolnai) [inline Sources/EduIDExpansion/Flows/CreateEduID/ViewModels/CreatePincodeAndBiometricAccessViewModel.swift]: Debug code

**Review by dzolnai — APPROVED** [review]

**Comment by phavekes** [comment]
Eduid changes do not belong in the tiqr core app

**Review by phavekes — CHANGES_REQUESTED** [review]
eduID specific changes do not belong in the tiqr core app, but in https://github.com/Tiqr/eduid-app-ios

**Comment by dzolnai** [comment]
@phavekes I just discussed this Yasser, and you are right, this is the wrong repo. Luckily this is not on the develop branch, so the tiqr app is not hindered by this yet. We merged this PR so we can show the functioning flows, but we will make sure that the code gets moved to the correct repo while we are continuing work on this.

**Pull request merged**